### DISCUSSION TITLE
Migration to OptiPlex does not detect objects 100ft away like old system.

### DISCUSSION THREAD

**Discussion by twister36** [blakeblackshear/frigate#9122] [Ask A Question]
So today I finished the migration from my Raspberry Pi to my new OptiPlex. Everything is "working", but I've noticed something strange.... 

When migrating I moved my configuration and Google Coral from the Raspberry Pi to the OptiPlex and change the necessary inputs for hardware acceleration changes. However, I noticed that on my driveway camera the new OptiPlex system would not detect cars about 100 feet away, but the old system (and actually a 3rd system) would detect the car driving by. Trying to troubleshoot, I removed, required_zone, moition masks with no luck. I then changed my detect input video from sub-stream to the primary stream and everything started to work.... ??

Any ideas why increasing the video quality caused it to detect the objects on the new system? Remember, same config, same Google Coral, same video source as the old system. Only difference is the computer hardware (Raspberry Pi 4 vs OptiPlex).

**Comment by NickM-27** [comment]
compare system page like inference speeds, detect fps, etc.

**Comment by twister36** [comment]
> compare system page like inference speeds, detect fps, etc.

Old system:
- 3 cameras
- Raspberry Pi 4 (8GB)
- Infra Speed: 12ms
- FPS: 5

New System:
- 9 cameras
- OptiPlex 5060 Micro, Core i7-8700 3.2GHz, 32GB RAM
- Infra Speed: 10ms
- FPS: 5

**Comment by twister36** [comment]
So after playing with this for hours, I think there are three things happening here: 

1. even though it's the same video source, same resolution (stream), same Google Coral, same config settings, the object detection on the new system is causing the object box to be slightly out of my required_zones, as when I remove the required zones it records, and when I alter the zone to match the new box detection box location, it records.

2. the Google Coral vs my Nvidia GPU (on a 3rd system) is scoring the objects less. Meaning that the Google Coral model is not able to score the same object at distance with a passing value like my Nvidia GPU TensorRT models is. However, I've noticed that TensorRT models used by the Nvidia cause many more false positives and object mismatches (i.e. Dog is seen as Cat, etc).

3. My old system running the same video, resolution, Google Coral and config was Frigate 12. The new system is Frigate 13-Beta7, so there may be other variables at hand here...

In any case, still working it out.

**Reply by NickM-27** [reply]
> My old system running the same video, resolution, Google Coral and config was Frigate 12. The new system is Frigate 13-Beta7, so there may be other variables at hand here...

that is a major difference, the object tracking is different as well as the fact that zones work differently by default
